schemas/EGA.analysis.json

```diff
@@ -5,9 +5,190 @@
+        "description": "Number of samples included in the analysis (i.e. analysed together). One sample corresponds to one biological replicate [EFO:0002091] (e.g. genetic content from a single cell, a tissue, buccal swab, etc.) from a single individual or from several individuals. Shall not be mistaken for technical replicates [EFO:0002090] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
+        "examples": [ 30 ]
+      },
+
+      "analysis_protocols": {
+        "type": "array",
+        "title": "Protocols related to an analysis",
+          "description": "Comprises metadata (e.g. Type of protocol) of a plan specification executed during an analysis. It shall have a sufficient level of detail and quantitative information to communicate it (and thus reproduce it) between investigation agents.",
+        "minItems": 1,
+        "additionalProperties": false,
+        "uniqueItems": true,
+        "items": {
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/protocols_object" 
+        }
+      },
+
+      "targeted_loci": {
+        "type": "array",
+        "title": "Loci of the targeted genomic feature",
+        "description": "Array of items that unambiguously define the loci of targeted genomic features in the analysis. For example, if the aim of the analysis was to detect variants in the human gene TAF1 and TP53, their identifiers will be expected in two items of this array.",
+        "minItems": 1,
+        "uniqueItems": true,
+        "additionalProperties": false,
+        "items": {
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/locus_identifier" 
+        }
+      },
+
+      "experimental_designs": {
+        "type": "array",
+        "title": "Experimental designs of the analysis [EFO:0001426]",
+        "description": "Array of experimental factors describing the method(s) of investigating particular types of research questions or solving particular types of problems. Each item of the array contain both the human readable ID (e.g. RNA stability design) and an optional label to qualify the term or provide additional context.",
+        "uniqueItems": true,
+        "additionalProperties": false,
+        "items": {
+          "title": "Experimental design item",
+          "description": "One of the experimental designs of the analysis.",
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/experimental_design"
+        }
+      },
+
+      "types_of_data": {
+        "type": "array",
+        "title": "Type of data [IAO:0000100]",
+        "description": "Type of raw data the analysis uses to obtain the processed files.",
+        "additionalProperties": false,
+        "uniqueItems": true,
+        "minItems": 1,
```

Comment:
Is this attribute required under the current schema? If not, I suggest not making it required here.